test/Concurrency/global_variables.swift

```diff
@@ -52,6 +54,11 @@ struct TestStatics {
   // expected-note@-1{{isolate 'wrapped' to a global actor, or convert it to a 'let' constant and conform it to 'Sendable'}}
 }
 
+public actor TestPublicActor {
```

Comment:
Nit: Do we have non-public actor test case? If not would be valuable to add?

Reply:
Good idea! I can add one 🙂